doc/releases/changelog-dev.md

```diff
@@ -341,6 +341,10 @@
   operators to the left, as is standard in quantum chemistry). 
   [(#5114)](https://github.com/PennyLaneAI/pennylane/pull/5114)
 
+* Multi-wire controlled `CNOT` and `PhaseShift` can now be decomposed correctly.
+  [(#5125)](https://github.com/PennyLaneAI/pennylane/pull/5125) 
```

Comment:
```suggestion
  [(#5125)](https://github.com/PennyLaneAI/pennylane/pull/5125/) 
```

I think the sphinx complaint is due to one link having a backslash and the other not.